DictionaryCompressor/DictionaryCompressor.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
Use all settings from our projects, e.g. EnableNullable and TreatWarningsAsErrors